README.md

```diff
@@ -70,6 +70,22 @@ transformations_over_time = train(transformations, X, y, splitter)
 pred = backtest(transformations_over_time, X, y, splitter)
 ```
 
+With the `train_evaluate` and `evaluate` helper function (requires [krisi](https://github.com/dream-faster/krisi)):
+
+```python
+from fold import train_evaluate
+X
+y = X.squeeze()
+
+splitter = ExpandingWindowSplitter(initial_train_window=400, step=400)
+transformations = [
+  DummyRegressor(strategy="constant", constant=0),
+  OnlyPredictions(),
+]
+scorecard, prediction, trained_transformations =  train_evaluate(transformations, X, y, splitter)
+
```

Comment:
I have added this section, but we can take it out if you'd like because it depends on krisi.